goagen/gen_app/generator.go

```diff
@@ -415,3 +417,81 @@ func (g *Generator) generateUserTypes() error {
 	}
 	return utWr.FormatCode()
 }
+
+// mediaTypeMetadataImports will grab all imports specified in a struct:field:type Metadata tag and
+// add them to the imports for the media_types.go file.
+func (g *Generator) mediaTypeMetadataImports(s []*codegen.ImportSpec) []*codegen.ImportSpec {
```

Comment:
Mainly that we only need one function, not a function for media type and another one for user type. One function that accepts an attribute works for both cases.